README.md

```diff
@@ -45,14 +43,12 @@ python setup.py install
 
 #### Advanced
 
-To leverage the power of FastFold, we recommend you build [Triton]() from source.
+To leverage the power of FastFold, we recommend you to install [Triton](https://github.com/openai/triton).
 
-**[NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 11.4 or above is needed.**
+**[NVIDIA CUDA](https://developer.nvidia.com/cuda-downloads) 11.3 or above is needed.**
```

Comment:
This line can be deleted for you have announced it in Installation.

Reply:
ok